benchmarks/rayleigh_taylor_instability_free_surface/plot_drip_depth.gnuplot

```diff
@@ -0,0 +1,28 @@
+reset
```

Comment:
Can you include a figure of this plot or the other one (or both) into the README file?

Reply:
I've added a figure for both scripts, but unfortunately I only ran one setup to completion, and it has a smaller timestep than in the plot scripts. It matches nicely though.